kin-sdk/src/androidTest/kotlin/kin/sdk/KinAccountIntegrationTest.kt

```diff
@@ -67,64 +70,57 @@ class KinAccountIntegrationTest {
         assertThat(status, equalTo(AccountStatus.NOT_CREATED))
     }
 
-    @Test
-    @LargeTest
-    fun getBalanceSync_AccountNotActivated_AccountNotActivatedException() {
-        val kinAccount = kinClient.addAccount()
-        fakeKinIssuer.createAccount(kinAccount.publicAddress.orEmpty())
-
-        expectedEx.expect(AccountNotActivatedException::class.java)
-        expectedEx.expectMessage(kinAccount.publicAddress.orEmpty())
-        kinAccount.balanceSync
-    }
-
     @Test
     @LargeTest
     @Throws(Exception::class)
-    fun getStatusSync_AccountNotActivated_StatusNotActivated() {
+    fun getBalanceSync_FundedAccount_GotBalance() {
         val kinAccount = kinClient.addAccount()
-        fakeKinIssuer.createAccount(kinAccount.publicAddress.orEmpty())
 
-        val status = kinAccount.statusSync
-        assertThat(status, equalTo(AccountStatus.NOT_ACTIVATED))
-    }
+        val latch = CountDownLatch(1)
 
-    @Test
-    @LargeTest
-    @Throws(Exception::class)
-    fun getBalanceSync_FundedAccount_GotBalance() {
-        val kinAccount = kinClient.addAccount()
+        var listenerRegistration : ListenerRegistration? = null
+        listenerRegistration = kinAccount.addAccountCreationListener {
+            listenerRegistration?.remove()
+            assertThat(kinAccount.balanceSync.value(), equalTo(BigDecimal("0.00000")))
```

Comment:
I think that all assertions should be in the main test thread (after the `latch.await` clause), it might get unnoticed by JUnit o.w.